### PR TITLE
Skip unsupported consumes

### PR DESCRIPTION
**Pull request by mxsc**

### CODE REVIEW

File: Sources/SwaggerSwift/parsePath.swift [hunk 1]
```diff
@@ -8,43 +8,57 @@ func parse(path: SwaggerSwiftML.Path, servicePath: String, swagger: Swagger, swa
     var functions = [NetworkRequestFunction]()
     var modelDefinitions = [ModelDefinition]()
     if let operation = path.get {
-        let result = parse(operation: operation, httpMethod: .get, servicePath: servicePath, parameters: parameters, swagger: swagger, swaggerFile: swaggerFile, verbose: verbose)
+        guard let result = parse(operation: operation, httpMethod: .get, servicePath: servicePath, parameters: parameters, swagger: swagger, swaggerFile: swaggerFile, verbose: verbose) else {
+            return (functions, modelDefinitions)
```
Comment (MadsBogeskov) [inline Sources/SwaggerSwift/parsePath.swift]: This doesn't seem right? if a http method fails shouldn't it continue to the next http method?

**Review by MadsBogeskov — CHANGES_REQUESTED** [review]
sd

File: Sources/SwaggerSwift/parsePath.swift (hunk 1 above)
Comment (mxsc) [reply Sources/SwaggerSwift/parsePath.swift]: just fixed that

File: Sources/SwaggerSwift/parsePath.swift [hunk 2]
```diff
@@ -7,44 +7,44 @@ func parse(path: SwaggerSwiftML.Path, servicePath: String, swagger: Swagger, swa
 
     var functions = [NetworkRequestFunction]()
     var modelDefinitions = [ModelDefinition]()
-    if let operation = path.get {
-        let result = parse(operation: operation, httpMethod: .get, servicePath: servicePath, parameters: parameters, swagger: swagger, swaggerFile: swaggerFile, verbose: verbose)
+    if let operation = path.get,
+       let result = parse(operation: operation, httpMethod: .get, servicePath: servicePath, parameters: parameters, swagger: swagger, swaggerFile: swaggerFile, verbose: verbose) {
         functions.append(result.0)
         modelDefinitions.append(contentsOf: result.1)
```
Comment (MadsBogeskov) [inline Sources/SwaggerSwift/parsePath.swift:13]:
```suggestion
       let (parsedFunctions, parsedDefinitions)  = parse(operation: operation, httpMethod: .get, servicePath: servicePath, parameters: parameters, swagger: swagger, swaggerFile: swaggerFile, verbose: verbose) {
        functions.append(parsedFunctions)
        modelDefinitions.append(contentsOf: parsedDefinitions)
```
just for readability

**Review by MadsBogeskov — CHANGES_REQUESTED** [review]
nitpick

File: Sources/SwaggerSwift/parsePath.swift (hunk 2 above)
Comment (mxsc) [reply Sources/SwaggerSwift/parsePath.swift]: totally agree, fixed

**Review by MadsBogeskov — APPROVED** [review]

**Pull request merged**